canonical/ubuntu.com
# [WD-10461] add lts to title

## Done

- Added the word "LTS" at the end of the title of the Thank You page for downloading desktop and server images. It will only add it if the release is actually LTS. For interim and non-LTS releases that word will not be added.

## QA

- Navigate to either /download/desktop or /download/server
- Click on download button
- On redirected Thank You page check that it shows "LTS" at the end of the title

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-10461

## Screenshots

![Screenshot 2024-04-17 at 17 46 14](https://github.com/canonical/ubuntu.com/assets/15943863/cf02bc7d-8b79-4c75-a3d7-cb61a4898838)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)
